dali/operators/generic/pad.cc

```diff
@@ -151,33 +152,36 @@ Samples are padded in the first axis to match a minimum extent of 4, and in the
     0.0f)
   .AddOptionalArg<int>("axes",
     R"code(Indices of the axes on which the batch samples will be padded.
-Indexes are zero-based with 0 being the outer-most dimension of the tensor.
-Arguments *axis_names* and *axes* are mutually exclusive.
-If *axes* and *axis_names* are empty or not provided, the output will be padded on all the axes)code",
-    std::vector<int>())
+
+Indexes are zero-based, with 0 being the outer-most dimension of the tensor. The ``axis_names``
```

Comment:
```suggestion
Indices are zero-based, with 0 being the outer-most dimension of the tensor. The ``axis_names``
```
We use this form two lines above, so let's be consistent.

Reply:
Done